### PR TITLE
website

### PR DESCRIPTION
**Pull request by blueUserRed**

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    <title>Document</title>
+    <link rel="stylesheet" href="./out/index.css">
+</head>
+<body>
+    
+    <header>
+        <img class="logo to-main-page" src="assets/logo_white.png" />
+        <a href="#" class="download">Download</a>
+        <a href="#" >We are open source!</a>
+        <a href="#" >Support us!</a>
```
Comment (GreenUserBlue) [inline website/index.html:15]: you forgot to add the hrefs (idk if this was intended since it is uncomplete)

File: website/index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                </p>
+            </article>
+            <footer>
+                <div class="links">
+                    <a class="to-imprint-page">IMPRINT</a>
+                    <a href="#">SUPPORT US</a>
+                    <a href="#">GITHUB</a>
```
Comment (GreenUserBlue) [inline website/index.html:73]: references missing

File: website/index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                </div>
+                <div class="copyright">Copyright © 2023 (Name)</div>
```
Comment (GreenUserBlue) [inline website/index.html:75]: name missing, prob bec we don't have one

File: website/index.html [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                </p>
+            </article>
+            <footer>
+                <div class="links">
+                    <a href="#">SUPPORT US</a>
+                    <a href="#">GITHUB</a>
+                </div>
+                <div class="copyright">Copyright © 2023 (Name)</div>
+            </footer>
```
Comment (GreenUserBlue) [inline website/index.html:125]: shoudln't the footer always be the same (it doesn't change for each page)

**Review by GreenUserBlue — APPROVED** [review]

File: website/index.html [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    <header>
+        <img class="logo to-main-page" src="assets/logo_white.png" />
+        <a href="#" class="download">Download</a>
+        <a href="#" >We are open source!</a>
+        <a href="#" >Support us!</a>
+        <div id="header-spacer"></div>
+        <img id="hamburger-menu" class="opens-navbar navbar-dependent" src="assets/hamburger.png" />
+    </header>
+
+    <nav class="navbar-dependent">
```
[the remark this answers is not in the document]
Comment (blueUserRed) [reply website/index.html:20]: identfiers with '-' are convention in css

File: website/index.html (hunk 1 above)
Comment (blueUserRed) [reply website/index.html:15]: these are placeholder links

File: website/index.html (hunk 2 above)
Comment (blueUserRed) [reply website/index.html:73]: these are placeholder links

File: website/index.html (hunk 3 above)
Comment (blueUserRed) [reply website/index.html:75]: placeholder

File: website/index.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+let curPageDependents;
+
+function main() {
+    doPageChangeListeners();
+    doNavbarListeners();
+    updateCurPageClasses();
+
+    // fix scroll offset when resizing
+    addEventListener("scroll", () => lastYScrollPosition = window.scrollY);
+    addEventListener("resize", () => fixScroll());
```
[the remark this answers is not in the document]
Comment (blueUserRed) [reply website/index.js:22]: its an anonymous function

**Pull request merged**